design/frontend/template/autocomplete/menu.phtml

```diff
@@ -7,9 +7,16 @@
         <div class="col3">
             <div class="other-sections">
                 <div class="aa-dataset-suggestions"></div>
-                <?php for ($i = 0; $i < 10; $i++): ?>
-                    <div class="aa-dataset-<?php echo $i; ?>"></div>
-                <?php endfor; ?>
+                <div class="aa-dataset-0"></div>
```

Comment:
That's the question where to stop. If we want to keep PHP or not. The same stuff as with translations in the templates ...